cfme/tests/services/test_provision_stack.py

```diff
@@ -285,8 +340,23 @@ def test_retire_stack(provider, create_template, catalog, request):
                   dialog=dialog_name, orch_template=template.template_name,
                   provider_type=provider.name)
     catalog_item.create()
-    stackname = "test" + fauxfactory.gen_alphanumeric()
-    stack_data = {'stack_name': stackname}
+    if provider.type == "azure":
```

Comment:
Seeing as the data seems to be cropping up everywhere, I propose that we DRY it out into a function, something like ```prepare_stack_data()``` which takes ```stackname``` as a required param, and then optionally other params and the provider object. In fact, thinking about it, it looks like stack_data could be a fixture as the contents look identical, then the logic of deciding on the stack_data is not inside the test, it is a lot easier to read.